packages/components/Button/src/CompoundButton/__snapshots__/CompoundButton.test.tsx.snap

```diff
@@ -1,91 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`CompoundButton default 1`] = `
```

Comment:
This isn't good, we'll lose coverage for render tree changes in these buttons. Can we change the platform that jest runs on so we can keep these snapshot tests? The jest.config file is currently configured to use Android as its platform.

Reply:
Have modified the jest testing platform to iOS for Button related components